src/browser/index.tsx

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-import './require';
```

Comment:
I'm assuming that removing this is covered by the updated rules in the webpack configuration?

Reply:
Other than this, everything looks great and I'm looking forward to merging this.

Reply:
Thanks. That is true. This weird hook is unnecessary right now. I only forgot to commit the removal of the actual file. Now it is removed.